packages/headless/src/features/configuration/analytics-params.ts

```diff
@@ -25,7 +25,9 @@ export const fromAnalyticsStateToAnalyticsParams = (
       ...(s.userDisplayName && {userDisplayName: s.userDisplayName}),
       ...(s.deviceId && {deviceId: s.deviceId}),
       ...(s.trackingId && {trackingId: s.trackingId}),
-      ...{capture: true},
+      ...{
+        capture: navigatorContext.capture ?? navigatorContext.clientId !== '',
```

Comment:
If the `clientId` is `''`, requests will fail, so this is a better fallback than `true` IMO.